presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -267,6 +293,113 @@ private Plan plan(String sql, LogicalPlanner.Stage stage)
         }
     }
 
+    private static void assertEqualPlans(Plan actual, Plan expected)
```

Comment:
Those tests are brittle. There's no guarantee that the two queries are going to produce exactly the same plan. There are too many factors involved. It's reasonable to check that the queries produce the same results, and you can do this by adding entries in AbstractTestQueries.

Also, we don't want to be carrying around and maintaining yet another framework for comparing plans. Please remove them.